cli/internal/run/run.go

```diff
@@ -1122,7 +1120,7 @@ func (ec *execContext) exec(ctx gocontext.Context, packageTask *nodes.PackageTas
 	if err := closeOutputs(); err != nil {
 		ec.logError(progressLogger, "", err)
 	} else {
-		if err = taskCache.SaveOutputs(ctx, progressLogger, targetUI, int(duration.Milliseconds())); err != nil {
+		if err = taskCache.SaveOutputs(ctx, progressLogger, prefixedUI, int(duration.Milliseconds())); err != nil {
```

Comment:
for what it's worth, `progressLogger` and `targetUI` (now `prefixedUI`) are being used for pretty much the same UI. The only difference is the `progressLogger` messages aren't printed without `--verbosity=3` flag (depending on level). 

As far as I could tell `progressLogger` logs aren't written to disk or anything either? In which case, maybe it makes sense to just remove them all together?

Reply:
I agree that `progressLogger` definitely needs to be cleaned up. It appears to be (mostly?) superfluous at this point. I don't have a good sense of how it came into existence though, so we should circle back to that.